### ISSUE TITLE
Doc: B.6. Instruction to Edit File Which Doesn't Exist

### ISSUE THREAD

**Issue by ode33** [stan-dev/cmdstan#685]
#### Summary:
Section B.6. of the CmdStan Guide currently reads:

"Users can customize how the tools are built by editing `make/local`
inside the `<cmdstan-home>` directory."

References to `make/local` continue throughout this section.

#### Description:
File `make/local` doesn't exist. An instruction to create such a file, in the case that it should be created, are missing. 

#### Current Version:
v2.19.0 (or latest)

**Comment by bob-carpenter** [comment]
Do we need more than "editing" changing to "editing (or creating)"?

**Comment by increasechief** [comment]
Not sure. No prior CmdStan releases has had the file. However files like `os_linux` where included up until recently.
```
##
# This file should modify the following variables
# as appropriate:
#   - CFLAGS
#   - CFLAGS_GTEST
#   - EXE
##

ifeq (g++,$(CC_TYPE))
  LDFLAGS += -lpthread
  CFLAGS_GTEST += -DGTEST_HAS_PTHREAD=0
  ifeq (4.8,$(CC_MAJOR).$(CC_MINOR))
    CFLAGS += -Wno-unused-local-typedefs
  endif
endif
ifeq (mingw32-g++,$(CC_TYPE))
  CFLAGS_GTEST += -DGTEST_HAS_PTHREAD=0
  LDLIBS_STANC += -static
  LDLIBS += -static
  LDFLAGS += -lwinpthread
  EXE = .exe
  WINE = wine
endif
ifeq (clang++,$(CC_TYPE))
#  CFLAGS += -stdlib=libc++
  CFLAGS_GTEST += -DGTEST_HAS_PTHREAD=0
  CFLAGS_GTEST += -Wno-c++11-long-long
  CFLAGS += -Wno-unused-function
  CFLAGS += -Wno-uninitialized
  CFLAGS += -Wno-c++11-long-long
endif
```

**Comment by increasechief** [comment]
Not certain, in the past releases `make/local` was always referred to but never exists. Otherwise it's seems simple to revise the wording to: "create and edit `make/local`". Or to include `local` as an empty template.

**Comment by bob-carpenter** [comment]
We can't check in an empty one without putting it in .gitignore, because otherwise it'll get overwritten.  So I think it'll be safer to just ask people to create one if they don't have one already.

**Comment by ode33** [comment]
Resolved by PR #687.
